src/include/units/random.h

```diff
@@ -0,0 +1,453 @@
+#pragma once
+
+#include <random>
+#include <functional>
+
+namespace units {
+
+template<typename Quantity>
```

Comment:
Please use concepts (`Quantity Q`) instead of pure `typename`.

Reply:
Oh, alright. Looked for Quantity under concepts but didn't notice it before.